doc/stdlib/dates.rst

```diff
@@ -136,6 +136,11 @@ alternatively, you could call ``using Dates`` to bring all exported functions in
    Returns a DateTime corresponding to the user's system
    time as UTC/GMT.
 
+.. function:: eps(::DateTime) -> Millisecond
+              eps(::Date) -> Day
+
+   Returns ``Millisecond(1)`` for ``DateTime`` values and ``Day(1)`` for ``Date`` values.
+
```

Comment:
You're correct that typing `?eps` doesn't bring up the docs for `Dates.eps`. Is that just something we need to fix on the `help` side of things? cc: @one-more-minute. Interestingly, doing `?Dates.eps` doesn't bring it up either.